astropy/time/tests/test_basic.py

```diff
@@ -2523,12 +2523,12 @@ def test_ymdhms_init_from_dict_scalar(kwargs):
     tm = Time(time_dict, **kwargs)
 
     assert tm == Time("2016-12-31T23:59:60.123456789")
-    for attr in time_dict:
+    for attr, expected in time_dict.item():
```

Comment:
This should be `.items()`, though!

Reply:
whoops. That's why manual fixes are always riskier. Thanks for catching it !